### PR TITLE
Add self as developer and member

### PR DESCRIPTION
**Pull request by shankgan**
The most common reason to send a PR to this repository is to join the Istio org.  If you are joining, please fill out this template.  If you are not, please **delete all the below**.

- [X] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [X] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [X] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [X] I have joined the [Istio discussion board](https://discuss.istio.io) and subscribed to the [Contributors](https://discuss.istio.io/c/contributors) category
- [X] I have joined [Istio's Slack workspace](https://slack.istio.io)

List your company name, or indicate Individual if you're not affiliated with a company:
Google

Provide a link to at least one PR that you have successfully pushed to one of the Istio repos:
https://github.com/istio/istio/pull/27418

### CODE REVIEW

File: org/teams.yaml [hunk 1]
```diff
@@ -57,6 +57,7 @@ teams:
       - SataQiu
       - sbezverk
       - sdake
+      - shankgan
```
Comment (howardjohn) [inline org/teams.yaml:60]: This list is for developers that are a maintainer of a working group, so I don't think you should add yourself here until you become a maintainer

Comment (howardjohn) [reply org/teams.yaml:60]: Which is hopefully soon :slightly_smiling_face:

Comment (shankgan) [reply org/teams.yaml]: oops..thanks for the clarification!

**Review by smawson — APPROVED** [review]

**Comment by sdake** [comment]
@howardjohn quick question about the developer label. I think we agreed in the past that developer == someone that develops code == member. Did that change to WG maintainer?

I think this causes a lot of frustration for our new contributors and if possible, it would be nice to collapse developer and member.

Welcome aboard @shankgan !

Cheers,
-steve

**Comment by howardjohn** [comment]
maintainer == someone that is a part of a working group. We can actually
automate this group to auto populate by the union of the working groups
just haven't done it yet

**Comment by howardjohn** [comment]
but also should collapse member and developer

**Pull request merged**